cocotb/_compat.py

```diff
@@ -0,0 +1,109 @@
+"""
```

Comment:
Please include the standard license header.

Maybe name this file "pycompat"? We might introduce compatibility code in the future which maintains compat with older cocotb versions.